Documentation/GettingStartedWithTheMRTK.md

```diff
@@ -38,28 +38,121 @@ To get started with the Mixed Reality Toolkit you will need:
 1. Go to the <a href="https://github.com/Microsoft/MixedRealityToolkit-Unity/releases" target="_blank">MRTK release page</a>.
 1. Under Assets, download
     - **Microsoft.MixedRealityToolkit.Unity.Foundation.unitypackage**
-    - (Optional) Microsoft.MixedRealityToolkit.Unity.Extensions.unitypackage
-    - (Optional) Microsoft.MixedRealityToolkit.Unity.Tools.unitypackage
-    - (Optional) Microsoft.MixedRealityToolkit.Unity.Examples.unitypackage
+    - (**_Optional_**) Microsoft.MixedRealityToolkit.Unity.Extensions.unitypackage
+    - (**_Optional_**) Microsoft.MixedRealityToolkit.Unity.Tools.unitypackage
+    - (**_Optional_**) Microsoft.MixedRealityToolkit.Unity.Examples.unitypackage
+    - (**_Optional_**, **_Experimental_**) Microsoft.MixedRealityToolkit.Unity.Providers.UnityAR.unitypackage
 
 The Mixed Reality Toolkit is now available for download on NuGet.org, for details see [MRTK NuGet Packages](MRTKNuGetPackage.md).
 
+#### Package contents
```

Comment:
Meta comment after looking at how this changes the flow of this document:

My understanding of the main purpose of this document is for someone truly getting started with the MRTK and providing the least-friction/least-resistance for getting running with examples in the editor.

The package contents is valuable, but it's currently a LOT of text and interrupts that main flow. i.e. first time users shouldn't have to read through all of this in order to get to the next section (which shows them how to import and then use the MRTK)

It would probably be more helpful to have this section much later in this doc (or in another page that's specific to the overall folder/package structure of the MRTK.

Reply:
I am happy to create a new file and link from here as well as the TOC

Reply:
I think that either of the other options (and possibly other options) would also work.

The main thing here is to avoid interrupting the crucial user flow.